SingularityService/src/main/java/com/hubspot/singularity/hooks/SnsWebhookManager.java

```diff
@@ -73,8 +72,7 @@ public SnsWebhookManager(@Singularity ObjectMapper objectMapper,
       this.snsClient = AmazonSNSClientBuilder.defaultClient();
     }
     this.webhookManager = webhookManager;
-    this.publishSemaphore = AsyncSemaphore.newBuilder(configuration::getMaxConcurrentWebhooks, executorServiceFactory.get("webhook-publish-semaphore", 1)).build();
-    this.publishExecutor = managedCachedThreadPoolFactory.get("webhook-publish");
+    this.publishExecutor = managedCachedThreadPoolFactory.get("webhook-publish", configuration.getMaxConcurrentWebhooks());
```

Comment:
Leaving a note here to double check our naming here, for both `managedCachedThreadPoolFactory` and `maxConcurrentWebhooks`

Reply:
did the rename for these. Going to leave the config var the same though for easier backwards compatibility. It is essentially still controlling the same thing, we are just being limited by the size of a fixed thread pool vs the async semaphore now. A future update could be to consolidate all the thread pool size configs in one spot